docs/edsl.md

```diff
@@ -266,8 +266,9 @@ float I[N], O[N / 2];
 for (int i = 0; i < N/2; ++i) {
     float curr_max = FLT_MIN;
     for (int j = 0; j < 2; ++j) {
-      curr_max = I[2 * i + j];
-      O.add_constraints({I[2 * i + j] > curr_max})
+        if (I[2 * i + j] > curr_max) {
+            curr_max = I[2 * i + j];
+        }
```

Comment:
Indent with 2 spaces instead of 4 to match the style of the rest of the file.

Reply:
So, I think I had messed up the indent in the previous commit. In this snippet, the indent seems to be with 4 spaces for the rest of the lines, except for the ones I had previously modified. Should I change everything to 2 instead?

Reply:
Fixed this.